CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Rename table names to follow the naming convention
```

Comment:
This is a breaking change, please prefix this with an all-caps, bold, **BREAKING CHANGE** notice.